packages/repository-tests/src/crud/relations/acceptance/belongs-to.inclusion-resolver.relation.acceptance.ts

```diff
@@ -165,6 +165,16 @@ export function belongsToInclusionResolverAcceptance(
       expect(toJSON(result)).to.deepEqual(toJSON(expected));
     });
 
+    it('throws error if the inclusion query contains a non-empty scope', async () => {
+      await orderRepo.create({
+        description: 'shiba',
+        customerId: existingCustomerId,
+      });
+      expect(
+        orderRepo.find({include: [{relation: 'customer'}], limit: 1}),
+      ).to.be.rejectedWith(`Invalid "scope is not supported`);
```

Comment:
Oh, so the initial version does not support `scope`?

Reply:
BTW, I was referring to the scope inside `include`.

Reply:
No. See https://github.com/strongloop/loopback-next/issues/3443